randovania/games/dread/json_data/Elun.txt

```diff
@@ -190,14 +192,10 @@ Extra - asset_id: collision_camera_003
   * Extra - right_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   > Door to Gyroscope Room
       Trivial
-  > Event - Elun - Ammo Recharge Room Blob
-      Diffusion Beam or Wave Beam
   > Ammo Recharge
       Trivial
   > Alcove
-      Diffusion Beam or Wave Beam
-  > Event - Release X
-      Diffusion Beam or Wave Beam
+      Diffusion Beam or Wave Beam or After Elun - Ammo Recharge Room Blob
```

Comment:
No, this should just be after the blob. And add the connection to the blob event back. Also, there is a template for using Diffusion or Wave